cluster-service/deploy/openshift-templates/arohcp-service-template.yml

```diff
@@ -222,8 +222,35 @@ parameters:
 - name: CS_SERVICE_PRINCIPAL_CREDS_BASE64
   description: Base64 encoded service principal credentials for CS. This is temporary for P1
   required: true
+- name: KEYVAULT_NAME
+  description: The name of the Key Vault where the first party credentials are stored.
+  value: "service-kv-aro-hcp-dev"
+- name: TENANT_ID
```

Comment:
Let's specify that this is the keyvault tenant id  in the name. 

Similar to my comment above, are all of our key vaults going to live in the same tenant? Or should we make this specific per environment. Maybe something like `AZURE_FIRST_PARTY_APPLICATION_KEYVAULT_TENANT_ID` or `AZURE_IDENTITY_CREDENTIALS_KEYVAULT_TENANT_ID` (where the arm helper identities can also use this tenant id)?

Reply:
Similar question from above. In addition, this tenant is the same as the one required for maestro and hypershift. Note that for maestro we create a new key vault to store its related secrets.

Reply:
Note that we also provide other tenant ids for other purposes. For example, in the current aro auth config, there is a tenant id there used for authenticating clients within CS. I think it should be clear in the name as to what the tenant will be used for to avoid confusion and prevent people using this for different purposes other than what it's intended for.

Reply:
TenantID should always be the same no matter what key vault you're accessing.  The only difference would be if you're using the 1p credential to auth into the customer's tenant, but that's not a configuration that's set as a runtime env var or command-line flag.

Reply:
Then in that case, maybe just `KEYVAULT_TENANT_ID` would suffice? or to keep it in line with the service keyvault, `SERVICE_KEYVAULT_TENANT_ID`? 

There is another TenantID that we'd need to set as an env var, its the tenant id of where CS is running on and will be added/documented as part of https://issues.redhat.com/browse/ARO-7425

Will the tenant where CS is running on and where the service keyvault is located always be the same?

Reply:
Tenant will always be the same in which the service keyvault exists and CS runs.  No need to make it specific to the key vault instance.